bin/wwwtxt

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+require "gli"
+require "wwwtxt"
+
+include GLI::App
+
+program_desc "Turn websites into plain text"
+
+version WwwTxt::VERSION
+
+desc "Download a single URL as plain text"
+flag [:o,:out_dir], default_value: File.join(ENV["HOME"], "texts")
```

Comment:
Subcommand flags are simply flags specified via the block argument of `#command`, ie

```ruby
command :some_command do |c|
  c.flag :something
  ...
```

https://github.com/davetron5000/gli/wiki/Subcommands